frontend/packages/employee-frontend/webpack.config.js

```diff
@@ -153,6 +153,10 @@ module.exports = function (env, argv) {
         '/api/internal': {
           target: process.env.API_PROXY_URL || 'http://localhost:3020'
         }
+      },
+      watchOptions: {
+        poll: 1000,
+        ignored: ["node_modules"]
```

Comment:
Is there a reason to _not_ ignore all of the `node_modules/` directories? With yarn workspaces a lot of the packages are actually in `frontend/node_modules/` instead of `frontend/packages/employee-frontend/node_modules/`
```suggestion
        ignored: /node_modules/
```

Reply:
I changed this as you suggested, seems to work just fine.